src/walletFactory.ts

```diff
@@ -37,7 +37,7 @@ export const WalletFromUsername = async ({
   return WalletFactory({ user, logger })
 }
 
-export const getFunderWallet = async ({ logger }) => {
-  const funder = await User.findOne({ username: yamlConfig.funder })
-  return WalletFactory({ user: funder, logger })
+export const WalletFromRole = async ({ logger, role }) => {
```

Comment:
we dont use it as a class so it is a function, please rename to `walletFromRole` or better `getWalletFromRole`